public/secure/login.html

```diff
@@ -2,7 +2,6 @@
 <html lang="en">
 
 <meta charset="UTF-8">
-<meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
I see this kept in other places. Leftover?

Reply:
I was suspecting this was causing the loop exceed but I was wrong and forgot to add it back. Thanks